docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -218,10 +218,10 @@ V&V Checks:
     - Incidence of ectopic pregnancies, abortion or miscarriage
     - get_draws(gbd_round_id=7, location_id=location_id, gbd_id_type='cause_id', gbd_id=[995,374], source='como', measure_id=6, metric_id=3, age_group_id=24, sex_id=2, year_id=2021, decomp_step='iterative')
     - incidence_c374 + incidence_c995
-    - These simulants will NOT continue in the model. Use the `total population incidence <Total Population Incidence Rate>`_ rate directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence.
+    - These simulants will NOT continue in the model. Use the `total population incidence <Total Population Incidence Rate>`_ rate directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence. 
   * - 1
     - % of simulants to attend each delivery facility type, based on their propensity
-    - At home (68.3%), in public/governmental health facility (26.6%), in private and/or NGO health facility (3.3%), and other (4.8%) 
+    - At home (68.3%), in public/governmental health facility (26.6%), in private for-profit health facility (2.5%), in private not-for-profit health facility (0.8%), and other (1.8%) 
```

Comment:
So it looks like you're splitting the private/NGO health facility category into two facility types, "private for-profit" and "private not-for-profit", right? Do you see a particular advantage to adding this further stratification?

Reply:
Ok, thanks, that's all good to know. Yes, this seems like a good topic to discuss tomorrow!